website/content/en/preview/concepts/metrics.md

```diff
@@ -94,9 +94,15 @@ Pod state is the current state of pods. This metric can be used several ways as
 ### `karpenter_cloudprovider_duration_seconds`
 Duration of cloud provider method calls. Labeled by the controller, method name and provider.
 
-### `karpenter_cloudprovider_instance_price_estimate`
+### `karpenter_cloudprovider_instance_type_memory`
+Memory, in Mib, for a given instance type.
+
+### `karpenter_cloudprovider_instance_type_price_estimate`
 Estimated hourly price used when making informed decisions on node cost calculation. This is updated once on startup and then every 12 hours.
 
+### `karpenter_cloudprovider_instance_type_vcpu`
```

Comment:
```suggestion
### `karpenter_cloudprovider_instance_type_cpu_cores`
```

`kube_state` calls them `cpu_cores` and `memory_bytes`. I wonder if we should align with that: https://github.com/kubernetes/kube-state-metrics/blob/main/docs/pod-metrics.md